doc/source/user/tutorial-svd.rst

```diff
@@ -122,146 +95,107 @@ can access this by using the syntax
            [ 85,  95, 112, ..., 121, 120, 120],
            [ 85,  97, 111, ..., 120, 119, 118]], dtype=uint8)
 
-From the output above, we can see that every value in ``img[:,:,0]`` is an
-integer value between 0 and 255, representing the level of red in each
-corresponding image pixel (keep in mind that this might be different if you
-use your own image instead of `scipy.misc.face`).
+Every value is an integer from 0 to 255, representing the level of red in each
+of the 768 * 1024 = 786,432 red pixels.
 
 As expected, this is a 768x1024 matrix::
 
     >>> img[:, :, 0].shape
     (768, 1024)
 
-Since we are going to perform linear algebra operations on this data, it might
-be more interesting to have real numbers between 0 and 1 in each entry of the
-matrices to represent the RGB values. We can do that by setting
+To perform linear algebra, let's rescale these values into real numbers
+between 0 and 1:
```

Comment:
In the original version, I had also written "perform linear algebra" and it was pointed out to me that it's not exactly a correct way to say it. BTW in math, at least in portuguese, we DO say "perform linear algebra" ;) So maybe "perform linear algebra operations" is better.

Reply:
This sounds like one of those grammatical conundra @rossbar was warning about in howto-docs.

I don't understand the reasoning behind appending the word "operations" nor its relevance if that's not how mathematicians talk. We perform calculus, not calculus operations. 

If anyone wants to defend "operations" I'm open to learning something.

Another option is to can change "perform" to "do" and walk away.

Reply:
I'd find it equally odd if you wrote "I'm about to perform calculus" and then proceeded to call `np.diff`. Just like you wouldn't typically say "I'm about to perform electrical engineering", or "I'm about to do some food science".

Linear algebra is a whole branch of mathematics: "one does not simply" apply it wholesale.